src/styles/variables.scss

```diff
@@ -37,6 +37,10 @@ $grey-7: #202020;
 $black: #000 !default;
 
 // Colors
+$color-text: $grey-7;
```

Comment:
The structure of this file is getting a little muddled now. Originally the variables were grouped into sections ("Greys" and "Colors") based on whether they had a hue or not and had non-semantic names within that. Now we have a section of generic greys followed by semantic colors.

We should come up with a plan to clarify things. Also, we need to think a bit about the wider group of products and how we'll update the palette elsewhere to bring it into line with what we have here, where it makes sense to do so.